services/autoscaling/src/simcore_service_autoscaling/models.py

```diff
@@ -1,9 +1,12 @@
 from dataclasses import dataclass, field
 from typing import Any, TypeAlias
 
-from aws_library.ec2.models import EC2InstanceData
+from aws_library.ec2.models import EC2InstanceData, EC2InstanceType, Resources
 from models_library.generated_models.docker_rest_api import Node
 
+AssignedTasksToInstance: TypeAlias = tuple[EC2InstanceData, list, Resources]
```

Comment:
suggestion: For all these tuples I would rather use `NamedTuple`. The advantage is that they are still tuples (and can used them as such) in addition to add structure by having every field a name. The only disadvantage is having to defined them but if you manage to compose them it can be overcome

Reply:
I will move to dataclass